### DISCUSSION TITLE
Creative Commons (CC BY-NC-SA 3.0 GR).

### DISCUSSION THREAD

**Discussion by sofiakolosidou** [community/community#123980] [New to GitHub]
Hi, I am new to github... I want to mark my work with Creative Commons (CC BY-NC-SA 3.0 GR).
Does anyone know how can I do that?
Thank you in advance!

**Comment by davevad93** [comment]
Hi @sofiakolosidou ,

There's no license template for `CC BY-NC-SA 3.0` on GitHub when choosing a license for a repo, so I think a workaround would be to create a blank license file and then copy the content directly from the official [CC BY-NC-SA 3.0 LEGAL CODE](https://creativecommons.org/licenses/by-nc-sa/3.0/legalcode.txt) page.
